content/css/concepts/inheritance/inheritance.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: 'Units'
+Description: 'The cascading in CSS. Styling that applies the the child element when the parent is the selector'
```

Comment:
I think the phrase "when the parent is the selector" is making the sentence less clear. Maybe something more like: Styling that is applied to a chilled element when they are applied to its parent.